tests/cache/tests.py

```diff
@@ -2162,6 +2162,31 @@ def test_304_response_has_http_caching_headers_but_not_cached(self):
         self.assertIn('Cache-Control', response)
         self.assertIn('Expires', response)
 
+    def test_cache_control_max_age(self):
+        view = cache_page(2)(hello_world_view)
+        request = self.factory.get('/view/')
+
+        # Request once
+        with mock.patch.object(time, 'time', return_value=1468749600):  # Sun, 17 Jul 2016 10:00:00 GMT
+            response = view(request)
+            response.close()
+            self.assertIn('Expires', response)
+            self.assertEqual('Sun, 17 Jul 2016 10:00:02 GMT', response['Expires'])
```

Comment:
Please swap the order in the assertions and put `response` on the left, `self.assertEqual(response['Expires'], 'Sun, 17 Jul 2016 10:00:02 GMT')`